source/reference/api/start.txt

```diff
@@ -151,12 +215,101 @@ Request
 Response
 ~~~~~~~~
 
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Example 3 - Start Sync from Replica Set to Sharded Cluster
+----------------------------------------------------------
+
+Request
+~~~~~~~
+
+.. literalinclude:: /includes/api/requests/start-rs-shard.sh
+   :language: shell
+
+Response
+~~~~~~~~
+
 .. literalinclude:: /includes/api/responses/success.json
    :language: json
    :copyable: false
 
 Behavior
 --------
 
+State
+~~~~~
+
 If the ``start`` request is successful, ``mongosync`` enters the
 ``RUNNING`` state.
+
+Shard Collections
+~~~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
```

Comment:
@jeff-allen-mongo: Good catch.  SC > RC is not supported (it throws an error), but I clipped it during edits.

The `sharding` option is required for RC > SC, since it needs special configuration to know which collections to shard and which keys to use in sharding.  Asymmetric SC > SC does not require special configuration, it just works on its own.

Reply:
Got it. Thanks for the context.